public/js/Calendar.js

```diff
@@ -283,17 +309,17 @@
 
       var startISO = moment(d.start).toISOString();
       var endISO = moment(d.end).toISOString();
-      var string = moment(d.start).format(self.format.preset) +' &ndash; '+ moment(d.end).format(self.format.preset);
+      var string = moment(d.start).format(self.format.preset) +' &ndash; <span class="sr-only">to</span>'+ moment(d.end).format(self.format.preset);
```

Comment:
Does the `&ndash;` character get read by screen readers? I'm over thinking this but I'm curious if it would put a long pause before it says "to" 😂

Reply:
@j0ntsang it doesn't, but actually using a `<span>` to break up the text does cause the screenreader to read it separately so there is a slight pause. Kinda awkward but figured it was still better than just reading the two dates